### DISCUSSION TITLE
Unable to import AriaButtonOptions

### DISCUSSION THREAD

**Discussion by JHunnicutt** [adobe/react-spectrum#4783] [General]
According to the [documentation](https://react-spectrum.adobe.com/react-aria/useButton.html) for the `useButton` hook, the props argument should be the type `AriaButtonOptions`. I see this interface exported from the [source file](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/button/src/useButton.ts), but it's completely inaccessible to me in the component file. I try and import it, but my auto-complete doesn't indicate that it can be downloaded, and if I do import it, I'm given the error that there's no exported member.

Curiously enough, I'm also given the error `"react-aria" has no exported member named 'AriaButtonOptions'. Did you mean 'AriaButtonProps'?` According to [this post](https://github.com/adobe/react-spectrum/pull/4600#issuecomment-1574385436), `AriaButtonProps` should have been removed, in favor of `AriaButtonOptions`.

What might be going on here with this interface, and what could explain the contradictory info that I'm seeing?

**Comment by reidbarber** [comment]
Looks like we may have forgotten to export that from the individual package and monopackage. Thanks! I've created a PR here: https://github.com/adobe/react-spectrum/pull/4784

**Reply by JHunnicutt** [reply]
Oh sweet! Happy to have pointed that out! Cheers!
